README.md

```diff
@@ -1 +1,36 @@
 # gazebo_ros2_control
+
+This repository contains the contents for testing gazebo_ros2_control
+
+It is running Gazebo and some other ROS 2 nodes.
+
+## Video + Pictures
+
+![](img/gazebo_ros2_control_position_pid.gif)
+
+## Running
+
+### Modifying or building your own
+
+```bash
+cd Docker
+docker build -t gazebo_ros2_control .
+```
+
+### To run the demo
+
+To run the demo we are going to use [rocker](https://github.com/osrf/rocker/) which is a tool to run docker images with customized local support injected for things like nvidia support. And user id specific files for cleaner mounting file permissions. You can install this tool with the following [instructions](https://github.com/osrf/rocker/#installation).
+
+The following command will launch Gazebo:
+
+```bash
+rocker --x11 --nvidia --name gazebo_ros2_control_demo gazebo_ros2_control:latest
```

Comment:
I'm using Rocker from debs on Bionic, and got this error:

`rocker: error: unrecognized arguments: --name`

Removing that argument allowed the command to continue:

```suggestion
rocker --x11 --nvidia gazebo_ros2_control:latest
```

But then when it launches, I always get:

```
[gzclient   -2] No protocol specified
[gzclient   -2] No protocol specified
[gzclient   -2] Aborted (core dumped)
```

I don't usually use rocker so I'm not sure what I'm going wrong.

Reply:
`--name` option it's included in `master` branch of rocker, it's not released yet. I can remove it and add it later

Reply:
Did you see any other errors?

Reply:
I added `verbose:=true` to the launch command and I see errors like:

```
[gzserver-1] [Err] [RenderEngine.cc:743] Can't open display: :1
[gzserver-1] [Wrn] [RenderEngine.cc:87] Unable to create X window. Rendering will be disabled
[gzserver-1] [Wrn] [RenderEngine.cc:287] Cannot initialize render engine since render path type is NONE. Ignore this warning ifrendering has been turned off on purpose.
...
[gzclient --verbose  -2] [Wrn] [GuiIface.cc:119] could not connect to display :1
[gzclient --verbose  -2] [Err] [GuiIface.cc:123] This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
[gzclient --verbose  -2] 
[gzclient --verbose  -2] Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.
[gzclient --verbose  -2] 
[gzclient --verbose  -2] [Msg] Could not load the Qt platform plugin "xcb" in "" even though it was found.
```

Probably some incompatibility between rocker and my NVIDIA drivers. Not a big deal. We may just want to add instructions for how to build and run without rocker. I'll leave that up to you.